lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -55,11 +55,11 @@ class CacheHeaders extends Audit {
    */
   static get defaultOptions() {
     return {
-      // 50th and 75th percentiles HTTPArchive -> 50 and 75
+      // 50th and 25th percentiles HTTPArchive -> 50 and 75, with p10 derived from them.
       // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2018_04_01_mobile?pli=1
-      // see https://www.desmos.com/calculator/8meohdnjbl
-      scorePODR: 4 * 1024,
```

Comment:
this one probably got the largest hypothetical shift, but it matters zero in reality


👍

Reply:
> this one probably got the largest hypothetical shift, but it matters zero in reality

yeah, I guess limiting it to multiples of 1024 messed with the precision, but in practice the difference in score from the current curve is a maximum of 0.001 (blue line). Changing this to `28.2 * 1024` drops that to about a fifth of the error (goldenrod? line), but agreed that it doesn't actually matter in this case.

Sorry people that were getting an 84.5 on this zero-weighted audit and are now going to get an 84.4 and get rounded down!


![error](https://user-images.githubusercontent.com/316891/81251704-e465f980-8ff1-11ea-8bb8-0e6f0ef2e329.png)